server/pkg/user/members.go

```diff
@@ -88,6 +88,11 @@ func (m *Members) ContainsUser(u ID) bool {
 	return ok
 }
 
+func (m *Members) ContainsIntegration(i IntegrationID) bool {
```

Comment:
```suggestion
func (m *Members) HasIntegration(i IntegrationID) bool {
```

Aldo do not forget to add a unit test